Documentation/Upgrade/ConvertGlobalExtensions/Index.rst

```diff
@@ -10,7 +10,7 @@ Convert global Extensions
 If you use global extensions, convert them to local ones.
 
 Global extensions used to be saved in folders inside
-:file:`typo3/ext/`, such as :file:`typo3/ext/realurl`. In current
```

Comment:
having realurl as example here is ok, right?

Reply:
I thought that it wasn't that timely anymore, since TYPO3 V9 has its own routing, it's more about this than having realurl or anything else.

If the idea is to provide an example extension that is located in :file:`typo3/ext/`, another extension might be more appropriate.